sdk/PowerBI.AspNet.Mvc/Html/ReportExtensions.cs

```diff
@@ -76,7 +76,7 @@ private static MvcHtmlString ReportHelper(this HtmlHelper htmlHelper, ModelMetad
             tagBuilder.MergeAttribute("powerbi-type", "report", true);
             tagBuilder.GenerateId(fullHtmlFieldName);
 
-            return new MvcHtmlString(tagBuilder.ToString(TagRenderMode.Normal));
+            return new HtmlString(tagBuilder.ToString(TagRenderMode.Normal));
```

Comment:
The change from `MvcHtmlString` to `HtmlString` is not technically necessary here, but as we aren't using the added members on `MvcHtmlString`, I changed to use its base class.